boards/silabs/radio_boards/slwrb4180a/board.yml

```diff
@@ -1,4 +1,11 @@
 boards:
   - name: slwrb4180a
+    vendor: silabs
     socs:
       - name: efr32mg21a020f1024im32
+    revision:
```

Comment:
SLWRB4180B is a completely different product from SLWRB4180A. Revisions should not be used, it must be a separate board.

Reply:
Thank you for your comments; I will add support for the separate boards soon.